### PR TITLE
usersurvey

### PR DESCRIPTION
**Pull request by shmcgavin**
@francopestilli @soichih  -- for now, this will need to live in the documentation but I wanted to put this content on a webpage we could link to on Twitter and in the newsletter.

### CODE REVIEW

**Comment by soichih** [comment]
I think the file name should not contain space (I think it would work??) but I'd rather not have them.

File: docs/User Survey [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<style> #disqus_thread, #__comments { display: none } </style>
+
+<h2> Take the brainlife.io user survey </h2>
+
+We are inviting all registered brainlife.io users take a survey about their experience with the platform. The survey is anonymous, unless users choose to share their contact information.
```
Comment (francopestilli) [inline docs/User Survey:5]: "users take a survey"=>"users <to> take a survey"

File: docs/User Survey [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+We are asking users to participate in this survey because understanding our users’ needs and experiences with brainlife.io will help our team improve the platform to meet the needs of all students and researchers who want to use it.
+
+This survey will run until July 1 and take approximately 5 to 10 minutes to complete. Before you begin, you will read a Study Information Sheet about this study.
```
Comment (francopestilli) [inline docs/User Survey:9]: This survey will take approximately 5 to 10 minutes to complete and run until July 1.

File: docs/User Survey [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+<h2> Take the brainlife.io user survey </h2>
+
+We are inviting all registered brainlife.io users take a survey about their experience with the platform. The survey is anonymous, unless users choose to share their contact information.
+
+We are asking users to participate in this survey because understanding our users’ needs and experiences with brainlife.io will help our team improve the platform to meet the needs of all students and researchers who want to use it.
+
+This survey will run until July 1 and take approximately 5 to 10 minutes to complete. Before you begin, you will read a Study Information Sheet about this study. 
+
+You can take the short survey here: [Click here to begin the survey](https://iu.co1.qualtrics.com/jfe/form/SV_0jozJ2vl9OkJSbb)
```
Comment (francopestilli) [inline docs/User Survey:11]: remove "short"

**Comment by soichih** [comment]
@shmcgavin Will you be able to rename / and make other changes @francopestilli suggested? Do you know how to amend a PR?

Also, I can go ahead and create a page under brainlife.io website - instead of adding this under /docs

**Comment by shmcgavin** [comment]
> @shmcgavin Will you be able to rename / and make other changes @francopestilli suggested? Do you know how to amend a PR?
> 
> Also, I can go ahead and create a page under brainlife.io website - instead of adding this under /docs

sorry, I thought @francopestilli's changes automatically changed the doc and I thought I changed the file name when I changed the title. Do I make the changes to the file name and his edits in my file? I can't figure out how to edit my file now -- do i convert the PR to a draft @soichih to be able to do that?

**Pull request merged**